page_object_ecomap/tests/TestBase.py

```diff
@@ -1,17 +1,17 @@
 import unittest
 import os
 from selenium import webdriver
+from page_object_ecomap.framework.Dictionary import DICTIONARY
 from page_object_ecomap.framework.Pages import *
-from page_object_ecomap.tests.TestData import TestData
 
 
 class TestBase(unittest.TestCase):
     driver = None
 
     @classmethod
     def setUpClass(cls):
-        cls.test_data = TestData(os.path.dirname(os.path.abspath(__file__)) + "/test_data_file.txt")
-        base_url = cls.test_data.get("base_url")
+        cls.test_data = DICTIONARY
```

Comment:
better switch to using
from page_object_ecomap.framework.Dictionary import DICTIONARY
on line 4, and in all other affected files